_redirects.yml

```diff
@@ -105,6 +105,8 @@
   - /go/deprecated/
 "/engine/reference/commandline/cli/#experimental-features":
   - /go/experimental/
+"/compose/compose-v2":
```

Comment:
oh! almost missed that (and we should have validation for this); could you add a trailing `/`? (I _think_ otherwise there will be a double redirect. `/go/compose-v1-eol/ -> /compose/compose-v2 -> /go/compose-v1-eol/`